python/python/tests/test_transaction.py

```diff
@@ -530,14 +531,17 @@ async def test_transaction_object_commands(
             assert response[0] == OK  # transaction.set(string_key, "foo")
             assert response[1] == "embstr"  # transaction.object_encoding(string_key)
             assert (
```

Comment:
nit: I bet if you remove the brackets for assert, it'll put this on one line

Reply:
the formatter doesn't let me if the comment is on the same line. I moved the comment above instead so that this isn't formatted as awkwardly